pandokia/check_expected.py

```diff
@@ -57,16 +61,18 @@
 def run(args) :
 
     spec = {
-        '-v' : '', 
-        '-h' : '=+',
-        '-p' : '=+',
-        '-c' : '=+',
+        '-v'  : '', 
+        '-h'  : '=+',
+        '-p'  : '=+',
+        '-c'  : '=+',
+        '-cm' : '=+',
```

Comment:
will bash take a multi-character flag with a single -?
We might need to make this -m for "custom metadata"

Reply:
I think it is better and safer to make it a single character. I will change them to -m instead.